code/datums/gameticker.dm

```diff
@@ -705,10 +706,12 @@ var/global/current_state = GAME_STATE_WORLD_INIT
 				player_dead = 1
 
 			//handle traitors
-			if (player.mind && ticker.mode.traitors.Find(player.mind))
+			if (player.mind && ticker.mode.traitors.Find(player.mind)) // Roundstart people get the full bonus
 				earnings = job_wage
 				bank_earnings.badguy = 1
 				player_dead = 0
+			else if (istype(player.loc, /obj/cryotron) || player.mind && all_the_baddies.Find(player.mind)) // Cryo'd or was a baddie at any point? Keep your shit, but you don't get the extra bux
+				player_dead = 0
```

Comment:
consider: renaming this variable since the name is not actually true to what the effect is

Reply:
Okay, var renamed to player_loses_held_item, since if its 1, the player loses their held item.